server/tests/queries.test.js

```diff
@@ -116,6 +124,13 @@ describe('Queries', () => {
       if (results.errors) {
         return expect(results.errors).toEqual(expected)
       }
+
+      // Format resulting article as how they should be stored in the cache
+      const cacheToExpect = new Map(results.data.articles.map(article => ([article.id, JSON.stringify({ ...article })])))
+
+      expect(cache).toEqual(cacheToExpect)
+      cache.clear()
```

Comment:
can you also add a test where the cache is already "warm" (already has some articles in it) and assert that the database api does not get called in that case?

Reply:
It doesn't look like this was addressed. Somewhere in the test there needs to be an assertion like

```javascript
expect(databaseApi.mock.calls).toContainEqual(...)
```

[Reference](https://jestjs.io/docs/en/mock-functions#custom-matchers)